spec/javascript/packages/document-capture/components/acuant-capture-spec.jsx

```diff
@@ -288,6 +288,35 @@ describe('document-capture/components/acuant-capture', () => {
       expect(document.activeElement).to.equal(button);
     });
 
+    it('shows a generic error if camera starts but cropping error occurs', async () => {
+      const trackEvent = sinon.spy();
+      const { container, getByLabelText, findByText } = render(
+        <AnalyticsContext.Provider value={{ trackEvent }}>
+          <DeviceContext.Provider value={{ isMobile: true }}>
+            <AcuantContextProvider sdkSrc="about:blank" cameraSrc="about:blank">
+              <AcuantCapture label="Image" name="test" />
+            </AcuantContextProvider>
+          </DeviceContext.Provider>
+        </AnalyticsContext.Provider>,
+      );
+
+      initialize({
+        start: sinon.stub().callsArgWithAsync(1, undefined, 'start-fail-code'),
```

Comment:
Hi @aduth, I am responding to feedback for this PR because Brittany will be out of office for a bit.  Do you happen to know how to trigger a crop error?  I wanted to follow up on your question.  Thanks!

Reply:
I'm not aware of how to trigger it. Part of the reason for the question was trying to understand where in [the Acuant source](https://github.com/Acuant/JavascriptWebSDKV11/blob/master/webSdk/AcuantCamera.js) this actually occurs, since the other error expected types were originally sourced by following all of the failure paths.